app/components/form/submit_button_component.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class Form::SubmitButtonComponent < ViewComponent::Base
+  def initialize(last_step:, current_step:)
+    if last_step == current_step
+      @text = "Submit"
+    else
+      @text = "Save and continue"
+    end

```

Comment:
This conditional should be it's own method. Probably something like `submit_text` or `button_text`